Articles/Packages/Extending/uploading-to-our.md

```diff
@@ -0,0 +1,86 @@
+
+To get started you first need to have an account on Our. Don't have an account? Head on over to the site and [register](https://our.umbraco.com/member/Signup).
+
+:::note
+Our uses a karma system where you get positive karma for answering forum posts, etc. To limit spam you will need to earn a little bit of karma before you can upload a package. Otherwise you can reach out to the [Umbraco support team](https://umbraco.com/contact-us/), who can help you out.
+:::
+
+### Creating your package
+
+Before you can upload your package, you first have to create your package. Read over the [Creating a Package](./creating-a-package.md) article.
+
+### Uploading the package to Our
+
+To get started head over to your [profile page](https://our.umbraco.com/member/profile/). It will tell you on this page if you have enough karma to upload your package, if you do then click `Add Package`.
+
+![Your packages profile page](images/PackagesPage.png)
+
+You may notice if you followed the [Creating a Package](../Creating-a-Package/) document that a lot of this information has already been entered. This isn't automatically pulled from the package file and will need to be entered again.
+
+The form will save each page as you go and the package won't become live until you finish the process and mark it live on the final page. This means you can come back and finish your package submission at any time.
+	
+## Package Details
+
+| Property | Value | Note |
+| -------- | ----- | ---- |
+| Title | My Package | The name of your package |
+| Current version | x.x.x | The version of your package. This will display on the front end in the button (regardless of file name) |
+| Package Category | E.g. Backoffice Extension | The category this package should be put into. |
+| Package Description | This package will extend the backoffice. | What your package does. Be clear and to the point about what your package does. |
+| License name | MIT License | As input in Umbraco |
+| License URL | http://opensource.org/licenses/MIT | As input in Umbraco |
+
+### Resources
+
+| Property | Value | Note |
+| -------- | ----- | ---- |
+| Package URL | https://umbraco.com | This url will be shown as the "packages url" when others install it. Will mostly be a Github repo or similar. |
+| Demonstration URL | https://umbraco.com | If you have a video, blog, audio description of how your package works in more depth, this is the place to advertise it. |
+| Source code URL | https://umbraco.com | A link to the source code of the package. |
+| NuGet package URL | | A link to the NuGet listing of your package if available. |
+| Bug tracking URL | | If you want people to submit bugs in a specific place, pop the link in here. |
+
+### Analytics and tracking
+
+| Property | Value | Note |
+| -------- | ----- | ---- |
+| Google Analytics code | UA-11111111-1 | This will tie in stats about your package into your Google Analytics account.
```

Comment:
```suggestion
| Google Analytics code | UA-11111111-1 | This will include in statistics from your package into your Google Analytics account.
```

Reply:
This sounds a bit off to me, maybe "This will include statistics from your package in your Google Analytics account."? 🤔

Reply:
Yes, sorry - I meant just "include".  Have updated the suggestion.